volatility3/framework/plugins/linux/hidden_modules.py

```diff
@@ -3,7 +3,8 @@
 #
 import logging
 from typing import List, Set, Tuple, Iterable
-from volatility3.framework import renderers, interfaces, exceptions, objects
+import volatility3.framework.symbols.linux.utilities.modules as linux_utilities_modules
+from volatility3.framework import renderers, interfaces, exceptions, Deprecation
```

Comment:
```suggestion
from volatility3 import framework
from volatility3.framework import renderers, interfaces, exceptions
```

Then use `framework.Deprecation`?  I'd expect David's future tool to flag this, so... might as well do it right first time round?

Reply:
To be fair, it's a little odd they're in a class anyway?  The class isn't versioned or anything, just seems to be used for grouping.  I'd almost prefer a `deprecation.py` and have them just live straight in there?

Reply:
Yea, the Deprecation is only previously used in `linux/symbols/__init__.py` so far, which is being changed in this PR anyway. I thought the capital D was odd so I can fix that. Also, can put into its own file under framework/. Your other feedback is good too. I like the date idea, so I will do that tomorrow as well. Brain is too burned for today, but will knock out first thing in the morning.